docs/getstarted/get_started_jupyter.md

```diff
@@ -0,0 +1,120 @@
+In this tutorial, we will run a Python [Jupyter Notebook](http://jupyter.org/) on FloydHub. Notebooks allow you to create and share documents that contain live code, visualizations and explanatory texts. This is an [example Notebook](https://github.com/floydhub/quick-start-pytorch/blob/master/mnist.ipynb). It is great for interactively writing and debugging your code and visualizing your results and data.
+
+Similar to the [Quick Start guide](./quick_start.md), we will train a CNN model for handwritten digit recognition using PyTorch and the MNIST database.
+
+If you are new to FloydHub, please ensure you have followed the [Quick Start guide](./quick_start.md) first. It introduces some important concepts used in this tutorial.
+
+## What we will accomplish in this guide
+
+- Learn how to create a new project on FloydHub
+- Start a Jupyter notebook on FloydHub's GPU server
+- Interactively run and debug your code
+- Mount datasets to use in your code
+
+## Quick preparation checklist
+
+- You must have a [FloydHub account](https://www.floydhub.com/login)
+- You must have `floyd-cli` [installed on your computer](../guides/basics/install.md)
+- You must [log in to FloydHub through the CLI](../guides/basics/login.md)
+
+## Setup
+
+### Create a new project
+For this tutorial, we will create a new Project. This project will be a collection of the jobs you run and their data, logs and results.
+
+To create a new Project, visit [www.floydhub.com/projects](https://www.floydhub.com/projects) and click on the "New Project" button on the top right hand corner.
+
+![Create new project](../img/create_new_project.jpg)
+
+We will name this project `mnist-pytorch`. Feel free to provide an apt description.
+
+The `Visibility` field indicates who can see your project. If you set it to `Public`, anyone can see your project, your code and data. If you are working on an open source project, this is a great way to share and contribute to the FloydHub community. If your code or data is proprietary, please select `Private`. This will ensure that only you and your team will have access to this project.
+
+### Get the code
+We will clone the [quick-start repository](https://github.com/floydhub/quick-start-pytorch) from Github to your local machine and run it on FloydHub. Run the `git clone` command in a brand new directory on your computer:
+
+```bash
+$ git clone https://github.com/floydhub/quick-start-pytorch.git
+Cloning into 'quick-start-pytorch'...
+$ cd quick-start-pytorch
+```
+
+```bash
+$ ls
+$ README.md mnist.ipynb
+```
+
+In this guide, we will use the `mnist_cnn.ipynb` Jupyter Notebook.
```

Comment:
I think its just `mnist.ipynb`

Reply:
It got changed when we updated the quickstart: https://github.com/floydhub/quick-start/blob/master/mnist_cnn.ipynb

Reply:
But this one is the pytorch quickstart: https://github.com/floydhub/quick-start-pytorch.git

Reply:
Good call. Fixed